README.md

```diff
@@ -118,7 +118,7 @@ The following attributes are a part of the Setup DSL <kbd>intellij { ... }</kbd>
 | Attributes | Values | 
 | :------------- | :--------- | 
 | <kbd>intellijRepository</kbd>, <kbd>jreRepository</kbd> - Urls of repositories for downloading IDE distributions and JetBrains Java Runtime. <br/><br/>|**Acceptable Values:** <br/><kbd>url</kbd><br/><br/>**Default Value:** <kbd>https://jetbrains.com/intellij-repository</kbd>, <kbd>https://cache-redirector.jetbrains.com/intellij-jbr</kbd>|
-| <kbd>pluginsRepositories { ... }</kbd> - Configure repositories for downloading plugin dependencies. <br/><br/>|**Configuration:** <br/><kbd>marketplace()</kbd> - use Maven repository with plugins listed in the JetBrains marketplace<br/><kbd>maven(repositoryUrl)</kbd> - use custom Maven repository with plugins<br/><kbd>custom(pluginsXmlUrl)</kbd> - use [custom plugin repository](https://www.jetbrains.com/help/idea/managing-plugins.html) <br/><br/>**Default Configuration:** <kbd>pluginsRepositories { marketplace() }</kbd>|
+| <kbd>pluginsRepositories { ... }</kbd> - Configure repositories for downloading plugin dependencies. <br/><br/>|**Configuration:** <br/><kbd>marketplace()</kbd> - use Maven repository with plugins listed in the JetBrains marketplace<br/><kbd>maven(repositoryUrl)</kbd> - use custom Maven repository with plugins<br/><kbd>maven { repositoryUrl }</kbd> - use custom Maven repository with plugins where you can configure additional parameters (credentials, authentication and etc.)<br/><kbd>custom(pluginsXmlUrl)</kbd> - use [custom plugin repository](https://www.jetbrains.com/help/idea/managing-plugins.html) <br/><br/>**Default Configuration:** <kbd>pluginsRepositories { marketplace() }</kbd>|
```

Comment:
README.md is generated, please edit corresponding file in `/docs` directory and run `:documentr` task for generating README

Reply:
Done